src/private/widgets/TabWidgetWidget.cpp

```diff
@@ -201,3 +205,34 @@ void TabWidgetWidget::updateMargins()
     m_cornerWidgetLayout->setContentsMargins(QMargins(0, 0, 2, 0) * factor);
     m_cornerWidgetLayout->setSpacing(int(2 * factor));
 }
+
+void TabWidgetWidget::contextMenuRequested(QPoint pos)
+{
+    auto *tabBar = QTabWidget::tabBar();
+    auto count = tabBar->count();
+    // We dont want context menu if there is only one tab
+    if (count <= 1) {
```

Comment:
just count() instead ? no need for the temporary variables

Reply:
Done